src/main/java/net/sacredlabyrinth/phaed/simpleclans/hooks/discord/DiscordHook.java

```diff
@@ -319,7 +396,17 @@ public void createChannel(@NotNull String clanTag)
             throw new CategoriesLimitException();
         }
 
-        createChannel(availableCategory, clanTag);
+        TextChannel textChannel = availableCategory.createTextChannel(clanTag).complete();
+
+        for (Map.Entry<ClanPlayer, Member> entry : discordClanPlayers.entrySet()) {
+            Clan entryClan = entry.getKey().getClan();
+            if (entryClan == null) {
```

Comment:
Possibly a unnecessary check, because the map is formed from clan#getMembers (so the clan exists).
(The IDE warning can be ignored)